frontend/src/features/admin-form/common/components/CollaboratorModal/CollaboratorModal.tsx

```diff
@@ -33,7 +35,7 @@ export const CollaboratorModal = ({
       >
         <ModalCloseButton />
         {isOpen && (
-          <CollaboratorWizardProvider>
+          <CollaboratorWizardProvider formIdContext={formId}>
```

Comment:
nit: suggest clearer name, `context` usually means a real react context, not the value context contains
```suggestion
          <CollaboratorWizardProvider formId={formId}>
```